docs/content/0.Config/5.Automate/1.Usage.md

```diff
@@ -1,3 +1,44 @@
+---
+title: Usage
+---
+
 # Automate
 
-Not yet documented ...
\ No newline at end of file
+The "Automate" feature can be used to **periodically request one or more PID**s and post the results to an **MQTT server**. Therefore, this feature requires [MQTT configured in WiCAN](../MQTT.md).
```

Comment:
It only posts the result to the mqtt broker if mqtt is enabled and by default it updates the http endpoint ```wican_1091a83e9e71.local/autopid_data```

Reply:
You mean there is an HTTP endpoint `/autopid_data` running on the WiCAN? What should it be used for and how?

Reply:
It's really there but for what?

![image](https://github.com/user-attachments/assets/fc4fcd39-ffd4-4d1a-866b-21d7fe9a6862)

Reply:
Presumably to do the same thing I would do mit MQTT Explorer: monitor the messages sent from the car.

Reply:
> You mean there is an HTTP endpoint `/autopid_data` running on the WiCAN? What should it be used for and how?

It's used for the mqtt integration 

> Presumably to do the same thing I would do mit MQTT Explorer: monitor the messages sent from the car.

Yes, that's right

Reply:
I did not know about the HTTP endpoint. I improved documentation and added pictures.

It would be very convenient  for users if you could add a "View Auto PID data" button somewhere on the "Automate" tab. It would basically redirect (or open a new tab) for `/autopid_data`